coffee-dto/coffee-dto-stub-gen/pom.xml

```diff
@@ -76,44 +82,112 @@
-					<protocArtifact>com.google.protobuf:protoc:${version.protoc.version}:exe:${os.detected.classifier}</protocArtifact>
-					<inputDirectories>
-						<!-- only own coffee proto files generate -->
-						<include>${project.build.directory}/proto-external</include>
-					</inputDirectories>
-					<includeMavenTypes>transitive</includeMavenTypes>
-					<includeDirectories>
-						<directory>${project.build.directory}/proto-external</directory>
-					</includeDirectories>
-					<outputTargets>
-						<outputTarget>
-							<type>java</type>
-						</outputTarget>
-						<outputTarget>
-							<type>grpc-java</type>
-							<pluginArtifact>io.grpc:protoc-gen-grpc-java:${version.io.grpc.version}:exe:${os.detected.classifier}</pluginArtifact>
-						</outputTarget>
-						<outputTarget>
-							<type>grpc-coffee</type>
-							<pluginArtifact>hu.icellmobilsoft.coffee:coffee-grpc-protoc:${project.version}</pluginArtifact>
-						</outputTarget>
-					</outputTargets>
-				</configuration>
-				<executions>
-					<execution>
-						<phase>generate-sources</phase>
-						<goals>
-							<goal>run</goal>
-						</goals>
-					</execution>
-				</executions>
-			</plugin>
 		</plugins>
 	</build>
+	
+    <profiles>
+        <profile>
+            <id>arm64-protoc-java-gen-with-preistalled-executable</id>
+            <activation>
+                <property>
+                    <!-- add this property to settings.xml -->
+                    <name>cpu.arch</name>
+                    <value>aarch_64</value>
+                </property>
+            </activation>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>com.github.os72</groupId>
+                        <artifactId>protoc-jar-maven-plugin</artifactId>
+                        <version>${version.protoc.maven.plugin}</version>
+                        <configuration>
+                            <protocArtifact>com.google.protobuf:protoc:${version.protoc}:exe:${os.detected.classifier}</protocArtifact>
+                            <inputDirectories>
+                                <!-- only own coffee proto files generate -->
+                                <include>${project.build.directory}/proto-external</include>
+                            </inputDirectories>
+                            <includeMavenTypes>transitive</includeMavenTypes>
+                            <includeDirectories>
+                                <directory>${project.build.directory}/proto-external</directory>
+                            </includeDirectories>
+                            <outputTargets>
+                                <outputTarget>
+                                    <type>java</type>
+                                </outputTarget>
+                                <outputTarget>
+                                    <type>grpc-java</type>
+                                    <pluginPath>/opt/homebrew/bin/protoc-gen-grpc-java</pluginPath>
```

Comment:
itt is jo lenne egy comment, ami a leirasra utal. Mivel ez fog elszalni itt fogjak keresni az okat. A fix utvonal, akkor is eleg eros, CI es egyeb gepeken nem nagyon betarthato, kellene egy kiskapu (pl. ENV parameter, default ertekkel).

Reply:
okes, ezt atirom valtozora, bar a brew az ide telepiti, es a nativ stubgent eddig nem nagyon tudtam mashogy telepiteni. (CI gepeken enm hiszem, hogy szukseges lenne ARM-en futtatni)